src/expandableCalendar/index.js

```diff
@@ -207,13 +207,13 @@ class ExpandableCalendar extends Component {
   }
 
   getMarkedDates() {
-    const {context, markedDates} = this.props;
+    const {context, markedDates, disabledByDefault} = this.props;
 
     if (markedDates) {
       const marked = _.cloneDeep(markedDates);
       if (marked[context.date]) {
         marked[context.date].selected = true;
-      } else {
+      } else if (!disabledByDefault) {
```

Comment:
I understand you don't want the month scroll to autoselect a disabled date, but what it should select instead? Or you mean to leave the calendar with no day selected?

Reply:
@Inbal-Tish I do think that if we set the prop disabledByDefault and the user changes the month in the calendar, the first day of the month (context.date) should NOT be marked if it's not part of the markedDates. We should leave the month with no day selected so the user knows there are no selectable days in that month. Hope this makes sense